dupserver.go

```diff
@@ -0,0 +1,56 @@
+package modgearman
+
+import (
+	"github.com/appscode/g2/client"
+)
+
+var dupjobsToSendPerServer = make(map[string](chan *answer))
+
+func initialiseDupServerConsumers(config *configurationStruct) {
+	if len(config.dupserver) > 0 {
+		for _, dupAddress := range config.dupserver {
+			logger.Debugf("creating dupserverConsumer for: %s", dupAddress)
+			dupjobsToSendPerServer[dupAddress] = make(chan *answer, config.dupServerBacklogQueueSize)
+			go runDupServerConsumer(dupAddress, dupjobsToSendPerServer[dupAddress], config)
+		}
+	}
+}
+
+func runDupServerConsumer(dupAddress string, channel chan *answer, config *configurationStruct) {
+	for {
+		item := <-channel
+		var error = sendResultDup(item, dupAddress, config)
```

Comment:
please use `:=` instead of `var`. That's more consistent with the rest of the codebase.